drivers/clock_control/clock_stm32l0_l1.c

```diff
@@ -41,6 +41,22 @@ static uint32_t get_pll_source(void)
 	return 0;
 }
 
+/**
+ * @brief get the pll source frequency
+ */
+__unused
+uint32_t get_pllsrc_frequency(void)
```

Comment:
```suggestion
static uint32_t get_pllsrc_frequency(void)
```

Reply:
This function is declared in clock_stm32_ll_common.h:51such that that the variant specific implementations can be used in clock_stm32_ll_common.c. While clock_stm32_ll_common doesn't use it in the case of stm32l0 and stm32l1 I couldn't mark it static just like that.

Reply:
Fair enough.